packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java

```diff
@@ -126,8 +128,8 @@ public void onMethodCall(MethodCall call, final Result result) {
             cameras.add(details);
           }
           result.success(cameras);
-        } catch (CameraAccessException e) {
-          result.error("cameraAccess", e.getMessage(), null);
+        } catch (Exception e) {
```

Comment:
In general it's better not to catch and rethrow `Exception`s broadly like this, it'll make stack traces hard to track down for people debugging the problem. Could you rewrite `handleException` to just take `CameraAccessException` and then continue to just catch that still?

Reply:
So the app crashes when registering the plugin and you attempt to catch `CameraAccessException` in `onMethodCall`. So far, nothing else has worked including the use of `@RequiresAPI` and `@TargetAPI` annotations. 

Weirdly enough, you can still catch `CameraAccessException`s in the inner `Camera` class. 

I haven't been able to find another solution.